templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.Mongo/server/mongo/mongo_service.py

```diff
@@ -14,22 +14,13 @@ def create():
     data = request.get_json()
     list_item = {'text': data['text']}
     created_item = list_items.insert_one(list_item)
-    return make_response(
-        jsonify(
-            {'_id': str(created_item.inserted_id), 'text': list_item['text']}
-        ), 201
-    )
+    json_response = jsonify({'_id': str(created_item.inserted_id), 'text': list_item['text']})
+    return make_response(json_response, 201)
```

Comment:
Suggestion: maybe we can add comments to these explaining what the codes mean? Would save some seconds having to search online what `201` is etc. Of course for famous codes like `404` it is not as useful.